compiler/damlc/daml-stdlib-src/DA/Internal/LF.daml

```diff
@@ -207,4 +207,5 @@ unpackPair : forall f1 f2 a1 a2. Pair f1 f2 a1 a2 -> (a1, a2)
 unpackPair = magic @"unpackPair"
 
 -- | Existential template type that can wrap an arbitrary template.
-data AnyTemplate
+data AnyTemplate = AnyTemplate
```

Comment:
Let's follow the common pattern in this module:
```suggestion
data AnyTemplate =
    -- | HIDE
    AnyTemplate Opaque
```

Reply:
What’s the point of adding a field to this constructor?

Reply:
ah nvm, got it.

Reply:
Semi-related, we shouldn't need the HIDEs in this module any more since the constructors are not exported, so damldocs now hides them automatically.